CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to rust-GameDig
+This project is very open to new suggestions, additions and/or changes, these 
+can come in the form of *discussions* about the project's state, *proposing a 
+new feature*, *holding a few points on why we shall do X breaking change* or
+*submitting a fix*.
+
+## Communications
+GitHub is the place we use to track bugs and discuss new features/changes,
+although we have a [Discord](https://discord.gg/NVCMn3tnxH) server for the
+community, all bugs, suggestions and changes will be reported on GitHub 
+alongside with their backing points to ensure the transparency of the project's
+development.
+
+## Issues
+When opening a new issue, make sure to fill the issue template. They are made
+to make the subject to be as understandable as possible, not doing so may result 
+in your issue not being managed right away, if you don't understand something
+(be it regarding your own problem/the issue template/the library), please state 
+so.
+
+## Development
+Note before contributing that everything done here is under the [MIT](https://opensource.org/license/mit/) license.
+### Naming
+Naming is an important matter, and it shouldn't be changed unless necessary.
+
+A game's identificator shall be created following these rules:
+1. Names composed of a maximum of two words will result in an id where the 
```

Comment:
Would be more complete if we mention the rule 4 exception here:
> maximum of two words (except when rule #4 applies)

Reply:
Do you mean this for the 2nd point? Rule #4 wouldn't apply to the 1st point cause that would end up being the same as #3.

Reply:
I meant that when rule 4 applies: there is a duplicate ID. Instead of using rule 2 to construct the ID with an ancronym we fall back to rule 1 to make it out of words. However rule 1 doesn't mention this and explicitly states the only case it applies is when there are <= 2 words.

Reply:
Oh, right, we could mention this.

Reply:
Looks right?

Reply:
lgtm!